rstudio-dependencies.txt

```diff
@@ -0,0 +1,6 @@
+git
+libclang-dev
+lsb-release
+psmisc
+sudo
+wget
```

Comment:
I think we can remove wget, see other comment

Reply:
I've removed `wget` as suggested.